example/ddf_app/Makefile.am

```diff
@@ -1,7 +1,7 @@
 include $(top_srcdir)/example/Makefile.inc
 
 bin_PROGRAMS = odp_ddf_app$(EXEEXT)
-odp_ddf_app_LDFLAGS = $(AM_LDFLAGS) -static -Wl,--export-dynamic
+odp_ddf_app_LDFLAGS = $(AM_LDFLAGS) $(DPDK_LIBS) -static -Wl,--export-dynamic
```

Comment:
pkgconfig won't be included into the test validation, they are two separate things.